tools/reccmp/reccmp.py

```diff
@@ -414,143 +416,119 @@ def can_resolve_register_differences(original_asm, new_asm):
 basename = os.path.basename(os.path.splitext(original)[0])
 pattern = '// OFFSET:'
```

Comment:
This can probably be removed?

Reply:
pattern yes. basename is used inside the loop to match the binary filename passed in against the module name (i.e. `// OFFSET: <this-part> 0x1234`) from the offset markers.

This could be related to the previous regression; reccmp was pulling in every .cpp and .h file in the entire repo, which included my unit test files added here. I changed it around to pull from the individual ISLE and LEGO1 directories instead, but for ISLE we _do_ want to compare the common functions from LEGO1. I switched it back and just modified all my test files instead, but maybe something is still wrong.

I am also a few commits behind the master so we are probably comparing the saved match percentage against stale code here.

Reply:
Here's why we have a regression:
```cpp
// OFFSET: ISLE 0x401c40
// OFFSET: LEGO1 0x10005530
inline virtual void SetAtomId(MxAtomId p_atomId) { this->m_atomId = p_atomId; } // vtable+20;
```
Two offset markers, and my parser isn't set up for that. I'll have to think about how to best handle this.